lib/HttpClient.ts

```diff
@@ -501,4 +535,10 @@ export class HttpClient implements ifm.IHttpClient {
 
         return bypass;
     }
+
+    private _performExponentialBackoff(retryNumber: number): Promise<void> {
+        retryNumber = Math.min(ExponentialBackoffCeiling, retryNumber);
+        const ms: number = ExponentialBackoffSlotTime/Math.pow(2, retryNumber);
```

Comment:
We should set some max cap on this.

Reply:
That's what the `Math.min` expression does. It sets the cap at `SlotTime*(2^Ceiling)`